src/artifacts.js

```diff
@@ -0,0 +1,58 @@
+const
```

Comment:
I don't know of a standard that promotes multiple variable decls like this.   What were you following here?

This one is particularly dense with the fat arrow funcs as well.